osu.Game/Screens/Play/SongProgress.cs

```diff
@@ -57,6 +60,24 @@ public IEnumerable<HitObject> Objects
             }
         }
 
+        private double strainStep;
+
+        public double StrainStep
+        {
+            get
+            {return strainStep;}
```

Comment:
Either use lambdas, or put brackets in the same lines with the `get`/`set` but with spaces between brackets